client/App.vue

```diff
@@ -453,15 +453,15 @@ header.top-bar
       font-weight: bold;
     }
   }
-
-body, main
+html, body
   height 100%
-main
-  position absolute
-  width 100%
+body, main
```

Comment:
Very strange to see CSS without brackets {}
I'd prefer to always have them.

Reply:
Checking if they had the styles of this section done this way on purpose or we can add them

Reply:
Done